share/man/man4/ix.4

```diff
@@ -29,31 +29,31 @@
 .\"
 .\" * Other names and brands may be claimed as the property of others.
 .\"
-.Dd May 17, 2023
-.Dt IXGBE 4
+.Dd October 26, 2024
+.Dt IX 4
 .Os
 .Sh NAME
-.Nm ixgbe
-.Nd "Intel(R) 10Gb Ethernet driver for the FreeBSD operating system"
+.Nm ix
+.Nd Intel 82598 10Gb Ethernet driver
```

Comment:
This change should be undone

Reply:
The Nd change? It improves consistency with the other drivers (see `apropos -s4 intel`), let the user search by model (`man -k 82598`) and doesn't line wrap on standard console.

Reply:
Sorry the line comment is not exact enough, the part I would remove is specifically the `82598`.  While it is true in some absolute sense, each of these cards is marketed independently and the rest of the family is closer affined to the 82599 (aka X520) while it does not add any particular value to the documentation.

Reply:
That sounds good, thank you.